packages/j/jodit.json

```
@@ -19,10 +19,9 @@
     "target": "jodit",
     "fileMap": [
       {
-        "basePath": "build",
+        "basePath": "",
         "files": [
-          "jodit*.+(css|js)",
-          "plugins/**/*.@(css|js)"
+          "!(examples|types)/**/*min.@(css|js)"
```

```suggestion
          "@(es2015|es2018|es2021|es2021.en|es5)/**/*.min.@(css|js)"
```
I would be much more comfortable with leaving the globstar in here if we specify the specific directories we want to pull from, rather than saying "everything everywhere except examples/types" which would leave us far more at risk of unexpected things being included down the road